src/viur/core/bones/select.py

```diff
@@ -44,14 +50,26 @@ def __init__(
             t.Callable[["SkeletonInstance", Self], t.Any],
         ] = None,
         values: dict | list | tuple | t.Callable | enum.EnumMeta = (),
+        translation_key_prefix: str | t.Callable[[Self], str] = "",
         **kwargs
     ):
-
+        """
+        Initializes a new SelectBone.
+
+        :param defaultValue: key(s) of the values which will be checked by default.
+        :param values: dict of key->value pairs from which the user can choose from
+            -- or a callable that returns a dict.
+        :param translation_key_prefix: A prefix for the key of the translation object.
+            It is empty by default, so that only the label (dict value) from the values is used.
+            A static string or dynamic method can be used (like `translation_key_prefix_bonename`).
```

Comment:
Doesn't this make sense in BaseBone as well?

Reply:
But other bones doesn't have `values`.